testing/features/step_definitions/navigation_steps.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+Given("the navigation is on the page") do
+  @component = Navigation::Default.new.tap(&:load)
+end
+
+Given("the navigation is on the page with header links") do 
+  @component = Navigation::WithHeader.new.tap(&:load)
+end
+
+Given("the navigation does not fit on the screen") do 
+  page.driver.browser.manage.window.resize_to(320, 568)
+end
+
+Given("the dropdown menu is already open") do
```

Comment:
Based on the way you have written this cuke (Gherkin step). This is really good. As it's more imperative. I'll put some suggestions to keep your other ones like this.